tools/apidoc/gen_toc.py

```diff
@@ -190,7 +190,8 @@
     'CA': 'Certificate',
     'listElastistorInterface': 'Misc',
     'cloudian': 'Cloudian',
-    'Sioc' : 'Sioc'
+    'Sioc' : 'Sioc',
+    'Diagnostics': 'Diagnostics'
```

Comment:
You can put the API under Misc, unless you think more APIs are going to join this section in apidocs.

Reply:
The retrieveDiagnostics file may join this section